src/lib/FlightTasks/tasks/Orbit/FlightTaskOrbit.cpp

```diff
@@ -52,6 +53,23 @@ FlightTaskOrbit::~FlightTaskOrbit()
 	orb_unadvertise(_orbit_status_pub);
 }
 
+void FlightTaskOrbit::handleParameterUpdate()
+{
+	updateParams();
```

Comment:
Let's call the base implementation in case it changes at some point.

Reply:
Done in https://github.com/PX4/Firmware/pull/12751/commits/2166b2f877185733fade41ec7906ae777b03ef6b